include/cassandra.h

```diff
@@ -2942,6 +2955,11 @@ cass_collection_append_decimal(CassCollection* collection,
                                size_t varint_size,
                                cass_int32_t scale);
 
+
+CASS_EXPORT CassError
+cass_collection_append_collection(CassCollection* collection,
```

Comment:
Seem to be missing a definition for this. Already talked about it. I assume it will be added with examples, when needed.